easybuild/easyconfigs/p/Python/Python-2.7.10-intel-2015b.eb

```diff
@@ -113,7 +113,7 @@ exts_list = [
     ('pytz', '2015.4', {
         'source_urls': ['https://pypi.python.org/packages/source/p/pytz'],
     }),
-    ('pandas', '0.16.1', {
+    ('pandas', '0.16.2', {
```

Comment:
don't do this please

if you really need the latest version of pandas, you should create a separate easyconfig for it